packages/rocketchat-livechat/assets/rocket-livechat.js

```diff
@@ -8,8 +8,9 @@
 // 		h.parentNode.insertBefore(j, h);
 // 	})(window, document, 'script', 'initRocket', 'http://localhost:5000/livechat');
 // </script>
+/* exported RocketChat */
 
-;var RocketChat = (function(w) {
+var RocketChat = (function(w) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313529457/issues/source?bid=3199634&fileBranchId=3199636#l13)